span.go

```diff
@@ -119,9 +119,13 @@ func (s *spanImpl) FinishWithOptions(opts opentracing.FinishOptions) {
 	if s.tracer.Options.DebugAssertUseAfterFinish {
 		// This makes it much more likely to catch a panic on any subsequent
 		// operation since s.tracer is accessed on every call to `Lock`.
+		pool := s.tracer.spanPool
 		s.reset()
```

Comment:
sorry, didn't see in time - yeah, I'll keep it in mind the next time I touch the code.